install/docker-compose.yml

```diff
@@ -22,6 +22,7 @@ services:
 
   web:
     image: ghcr.io/georiviere/georiviere-admin:latest
+    user: $UID:$GID
```

Comment:
$UID and $GID are not automaticaly mapped by all system, so using these env varables should probably be wrapped in .env file